server/testbed/README.md

```diff
@@ -0,0 +1,27 @@
+# Virtual testbed setup
+
+This testbed allows you to generate a linux distribution image (ie. archlinux, fedora, debian) with ultrablue server
+configured in it.
+
+Your machine needs a **TPM2** and **bluetooth** devices.
```

Comment:
Even with swtpm, it's not a full emulation and you still need a hw one?

Reply:
My bad, changed it to swtpm requirement